express/routes/players.js

```diff
@@ -7,8 +7,11 @@ const validations = require('../validations/players')
 
 const router = express.Router();
 
-router.get('/', async (req, res) => {
-	const players = await playersService.getAll();
+router.get('/', validate(validations.get), async (req, res) => {
+	const players = await playersService.getAll(
```

Comment:
How about extracting props here to consts and then passing them to the service?

Reply:
Also please make them named args, there will be more params in the future and not all of them will be present, it's better to pass them as an object